package/piksi_system_daemon/piksi_system_daemon/src/whitelists.c

```diff
@@ -56,68 +56,226 @@ static port_whitelist_config_t port_whitelist_config[PORT_MAX] = {
   [PORT_UART0] = {
     .name = "uart0",
     .wl = "72,74,117,65535"
+    /*  This filter represents the messages a base station must output for a
+        rover to be able to calculate differential GNSS positions against it.
+        This filter removes all other messages to save bandwidth. Ephemeris
+        messages used to be included in this filter, but their bursty nature
+        tended to swamp communication links so they were taken out.
+        MsgBasePosECEF                72
+        MsgObs                        74
+        MsgGloBiases                 117        
+        MsgHeartbeat               65535
+    */
+
   },
   [PORT_UART1] = {
     .name = "uart1",
-    .wl = "23,29,65,72,74,80,117,134,136,144,163,165,166,167,171,175,181,185,187,188,257,258,259,520,522,524,526,527,528,1025,2304,2305,2306,65280,65282,65535"
```

Comment:
175 is being dropped by this change, which is needed for the recent settings write response changes